cmd/determinize-prow-config/main_test.go

```diff
@@ -70,3 +76,109 @@ func TestDeduplicateTideQueries(t *testing.T) {
 		})
 	}
 }
+
+func TestShardProwConfig(t *testing.T) {
+	testCases := []struct {
+		name string
+		in   *config.ProwConfig
+
+		expectedConfig     config.ProwConfig
+		expectedShardFiles map[string]string
+	}{
+		{
+			name: "Org and repo config get written out",
+			in: &config.ProwConfig{
+				BranchProtection: config.BranchProtection{
+					Orgs: map[string]config.Org{
+						"openshift": {
+							Policy: config.Policy{Protect: utilpointer.BoolPtr(false)},
+							Repos: map[string]config.Repo{
+								"release": {Policy: config.Policy{Protect: utilpointer.BoolPtr(false)}},
+							},
+						},
+					},
+				},
+			},
+
+			expectedShardFiles: map[string]string{
+				"openshift/_prowconfig.yaml": strings.Join([]string{
```

Comment:
Why not back-tick string literals for this?

Reply:
Not a super strong opinion one way or another, but I find backtick literals more annoying to read because the second line onwards starts at the beginning of the file without any indent. `cmp` spat this out so I didn't hand write it and it looks quite nice to me.